pages/agent/v3/help/_start.md

```diff
@@ -85,7 +85,9 @@ $ buildkite-agent start --token xxx
 <tr id="no-git-submodules"><th><code>--no-git-submodules </code> <a class="Docs__attribute__link" href="#no-git-submodules">#</a></th><td><p>Don't automatically checkout git submodules [$BUILDKITE_NO_GIT_SUBMODULES, $BUILDKITE_DISABLE_GIT_SUBMODULES]<br /><strong>Environment variable</strong>: <code>$BUILDKITE_NO_GIT_SUBMODULES</code></p></td></tr>
 <tr id="no-feature-reporting"><th><code>--no-feature-reporting </code> <a class="Docs__attribute__link" href="#no-feature-reporting">#</a></th><td><p>Disables sending a list of enabled features back to the Buildkite mothership. We use this information to measure feature usage, but if you're not comfortable sharing that information then that's totally okay :)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_NO_FEATURE_REPORTING</code></p></td></tr>
 <tr id="allowed-repositories"><th><code>--allowed-repositories value</code> <a class="Docs__attribute__link" href="#allowed-repositories">#</a></th><td><p>A comma-separated list of regular expressions representing repositories the agent is allowed to clone (for example, "^git@github.com:buildkite/.*" or "^https://github.com/buildkite/.*")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_ALLOWED_REPOSITORIES</code></p></td></tr>
-<tr id="allowed-plugins"><th><code>--allowed-plugins value</code> <a class="Docs__attribute__link" href="#allowed-plugins">#</a></th><td><p>A comma-separated list of regular expressions representing plugins the agent is allowed to use (for example, "^buildkite-plugins/.*$" or "^/var/lib/buildkite-plugins/.*")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_PLUGINSS</code></p></td></tr>
+<tr id="enable-environment-variable-allowlist"><th><code>--enable-environment-variable-allowlist </code> <a class="Docs__attribute__link" href="#enable-environment-variable-allowlist">#</a></th><td><p>Only run jobs where all environment variables are allowed by the allowed-environment-variables option, or have been set by Buildkite<br /><strong>Environment variable</strong>: <code>$BUILDKITE_ENABLE_ENVIRONMENT_VARIABLE_ALLOWLIST</code></p></td></tr>
+<tr id="allowed-environment-variables"><th><code>--allowed-environment-variables value</code> <a class="Docs__attribute__link" href="#allowed-environment-variables">#</a></th><td><p>A comma-separated list of regular expressions representing environment variables the agent will pass to jobs (for example or "^MYAPP_.*$"). Environment variables set by Buildkite will always be allowed. Requires --enable-environment-variable-allowlist to be set<br /><strong>Environment variable</strong>: <code>$BUILDKITE_ALLOWED_ENVIRONMENT_VARIABLES</code></p></td></tr>
```

Comment:
It looks like there's something missing here:

> (for example or "^MYAPP_.*$")

It's either adding another value as an example, or simplifying it to `(for example, "^MYAPP_.*$")`

Reply:
Good point. Looks like it's like that in the original: https://github.com/buildkite/agent/pull/2539/files#diff-978b503ab54eee245e9d9de4050c039219fe264fc02958e53a1d36e8ea097a20R573

I'll fix it in the agent then over here.

Reply:
https://github.com/buildkite/agent/pull/2598